packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission.jsx

```diff
@@ -159,6 +159,7 @@ const POISubmission = ({
                             is_from_external={is_from_external}
                             setIsCfdPoiCompleted={setIsCfdPoiCompleted}
                             allow_poi_resubmission={allow_poi_resubmission}
+                            handleViewComplete={handleViewComplete}
```

Comment:
Should we add this new change in proof-of-identity-submission-for-mt5

```
  case service_code.manual:
                return <Unsupported is_mt5 handlePOIforMT5Complete={handlePOIComplete} />;
      
```

Reply:
Thanks for noticing @amina-deriv , I'll take a look at this

Reply:
@amina-deriv I did check & it's not needed as we've a condition to check if it's is_mt5 in details-component so it's fine
`handleViewComplete={is_mt5 ? handlePOIforMT5Complete : handleComplete}`